patterns/1-initial/modular-code.md

```diff
@@ -8,54 +8,66 @@ The lack of modularization in the software architecture prevents reuseability, a
 
 ## Problem
 
-Engineering does not want to spend the extra resources needed to develop modular components. This applies to managers and developers alike. As a result, fewer components end up being reused, leading to duplication of work.
+* With rare exceptions, monolythinc code is difficult to contribute to.
+* The same functionality is reimplemented redundantly causing
+  * Waste of unneeded re-design and re-development
+  * Duplicated mainteinance effort
 
 ## Context
 
-* There is no corporate mandate for modularized code.
-* This is a new product/new code, not a legacy product/code.
+* There might be potentially reusable functionality buried in monolythic code.
```

Comment:
monolithic

Reply:
Hi @NewMexicoKid. Not sure if you have seen that GitHub feature yet.

When commenting you can hit the ![Screenshot 2022-10-26 at 18 09 58 ](https://user-images.githubusercontent.com/163029/198078602-5dd1f1b0-e9cc-4c6f-b80b-e25748e7def8.png)  symbol at the top of the comment box and suggest specific changes.
That allows the authors to clearly understand what you are referring to. Plus if the change doesn't need further conversation they can just click "commit" and are done :)

Below a screenshot of the button that I am referring to:

![Screenshot 2022-10-26 at 18 08 05 ](https://user-images.githubusercontent.com/163029/198078268-07378837-6f99-4cb5-9dab-c200d0b587bb.png)